modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminModal.js

```diff
@@ -38,35 +38,30 @@ const TranslationAdminModal = ({
 	const [activeLanguageIds, setActiveLanguageIds] = useState(
 		initialActiveLanguageIds
 	);
-	const [lastDeletedLanguageId, setLastDeletedLanguageId] = useState(null);
 	const [visible, setVisible] = useState(initialVisible);
 
+	const onCancel = () => {
+		setVisible(false);
+		onClose([...initialActiveLanguageIds]);
```

Comment:
Instead calling `onClose` with the `initialActiveLanguageIds` we can first call `setActiveLanguageIds` with the initial values and then call `onClose` with `activeLanguageIds`, in this way we have both values synchronized.